libs/crypto/merkle.cpp

```diff
@@ -0,0 +1,273 @@
+#include <sstream>
+#include <iostream>
+
+#include "merkle.h"
+using IROHA_CRYPTO_HPP::Integer;
+
+#define BLOCKSIZE 16
+#define DIGESTSIZE 32
+
+using namespace std;
```

Comment:
oh, quite dangerous, don't use this please

Reply:
Okay, got it. Any alternative for this?

Reply:
if you need something specific, for example `std::vector`

use:
```
using std::vector
```

Reply:
Yes @Warchant , I think `#include <vector>` will do this.